### PR TITLE
Mobile menu fixes

### PR DESCRIPTION
**Pull request by chrisopedia**
Why is this pull request necessary:

1. allows the user to use the menu on desktop
2. no longer tied to a specific device

Changes proposed in this pull request:

- remove the media queries around the menu
- rename it to `menu-slide`

Screenshot of docs:

no change in current docs

Notify or mention any users: @jgallen23

### CODE REVIEW

File: styles/helpers/menu-slide.css [hunk 1]
```diff
@@ -16,8 +16,22 @@ title: Mobile Menu
   </div>
 </div>
 ```
+```example:html
+<div class="menu-slide left">
```
Comment (jgallen23) [inline styles/helpers/menu-slide.css]: I think it's too risky to add the modifier as just `left`.  What if we add a helper class down the road that is just `left`?

**Comment by jgallen23** [comment]
docs look a little broken
screenshot 2016-09-11 at 7 21 49 pm

**Comment by jgallen23** [comment]
implementing looks a little jacked too

screenshot 2016-09-11 at 7 25 19 pm

**Comment by jgallen23** [comment]
can we make the menu take up space?
screenshot 2016-09-11 at 7 28 04 pm

**Comment by chrisopedia** [comment]
@jgallen23 regarding the "taking up space" I updated the docs for you so you can see how that's achieved without touching the menu code.  Essentially we're using the grid to give it space which allows our grid to power all layouts.

```
<div class="col-9"><h1>ClientKit</h1></div>
<div class="col-3">
  <div class="menu-slide">
    <input id="header-menu" class="menu-btn" type="checkbox" />
    <label for="header-menu"></label>
    <div class="menu-container text-right">
      <nav class="menu-header-container">
        <a href="">Menu 1</a>
        <a href="">Menu 2</a>
        <a href="">Menu 3</a>
        <a href="">Menu 4</a>
      </nav>
    </div>
  </div>
</div>```

**Comment by jgallen23** [comment]
@chrisopedia I don't want the menu on the right.  I want it to the left of the logo

File: styles/helpers/menu-slide.css (hunk 1 above)
Comment (chrisopedia) [inline styles/helpers/menu-slide.css]: changed to `menu-left`

**Pull request merged**